### PR TITLE
Display .post-create message after install

### PR DESCRIPTION
**Pull request by lorensr**
## What was changed

- Display sample's `.post-create` message after install (see https://github.com/temporalio/samples-typescript/pull/43)

image

Also implemented the improvements from this issue: https://github.com/temporalio/sdk-typescript/issues/333#issuecomment-955113742

- add a check that covers when the provided repo has changed names/orgs:

```
Downloading files from repo https://github.com/lorensr/samples-node/tree/post-create/hello-world-mtls. This might take a moment.

We were unable to download and extract the provided project.
 Archive URL: https://codeload.github.com/lorensr/samples-node/tar.gz/post-create
 Archive path: samples-node-post-create/hello-world-mtls
 Sometimes this is due to the repo name changing. If that's the case, try using the new repo URL.
```

- print a list of samples and link to samples-typescript

image

## Why?

- Repetitive and sometimes incorrect post-create messages: https://temporaltechnologies.slack.com/archives/C01FT8U10GK/p1635207289037300
- Unclear error messages

## Checklist

1. Closes https://github.com/temporalio/sdk-typescript/issues/333

2. How was this tested: 

```
lerna bootstrap
cd packages/create-project
npm run build
./cli.js myProj -s https://github.com/lorensr/samples-node/tree/post-create/hello-world-mtls
```

3. Any docs updates needed?
No

### CODE REVIEW

**Comment by lorensr** [comment]
(this code shouldn't be published to npm until this is merged: https://github.com/temporalio/samples-typescript/pull/43)

**Comment by bergundy** [comment]
Thanks @lorensr, I'll review in the morning

File: packages/create-project/src/create-project.ts
Comment (bergundy) [inline packages/create-project/src/create-project.ts:192]: We should not fail the process if this file does not exist IMHO because this tool accepts arbitrary URLs.

File: packages/create-project/src/create-project.ts
Comment (bergundy) [inline packages/create-project/src/create-project.ts:199]: I'd prefer reading the file contents using node and using chalk-template directly instead of spawning a process: https://github.com/chalk/chalk-template

File: packages/create-project/src/helpers/samples.ts
Comment (bergundy) [inline packages/create-project/src/helpers/samples.ts:113]: I don't know if I'd want to bother with stating the repo change cause, we're not expecting to change the repo name anymore 😬

**Review by bergundy — COMMENTED** [review]
Just some minor comments.

File: packages/create-project/src/create-project.ts
Comment (lorensr) [reply packages/create-project/src/create-project.ts:192]: Oops! 😄

File: packages/create-project/src/helpers/samples.ts
Comment (lorensr) [reply packages/create-project/src/helpers/samples.ts:113]: This is only when they provide their own URL, and it's the only situation I know of in which we'd reach this code path. (the `URL/package.json` is checked earlier)

File: packages/create-project/src/create-project.ts
Comment (lorensr) [reply packages/create-project/src/create-project.ts:199]: It's not currently published to npm, but it looks like it [might be soon](https://github.com/chalk/chalk-template/pull/3#pullrequestreview-794649035). After this is merged I could open a draft PR based on the future package?

Comment (bergundy) [reply packages/create-project/src/create-project.ts]: So until then please avoid the shell expansion.
You should be able to feed the template directly as an arg here using:
```ts
await fs.readFile(messageFile, 'utf8')
```

Comment (lorensr) [reply packages/create-project/src/create-project.ts]: Done in https://github.com/temporalio/sdk-typescript/pull/349/commits/eb64788a1becec562d54d545ff3af6166f45d4e0

File: packages/create-project/src/create-project.ts
Comment (bergundy) [inline packages/create-project/src/create-project.ts:207]: I'm definitely not blocking on this, but you might as well read the file as 'utf8' and get a string back.

File: packages/create-project/src/create-project.ts
Comment (bergundy) [inline packages/create-project/src/create-project.ts:208]: Just verifying, you tested this right?

File: packages/create-project/tsconfig.json
Comment (bergundy) [inline packages/create-project/tsconfig.json:9]: nit: we don't need the types here, the package would probably include types when it gets published.

**Review by bergundy — APPROVED** [review]
Nothing blocking, thanks!

File: packages/create-project/src/create-project.ts
Comment (lorensr) [reply packages/create-project/src/create-project.ts:208]: yep!

**Pull request merged**
